cypress/local-only/tests/integration/messages/messages.cy.ts

```diff
@@ -92,79 +92,18 @@ describe('Messages', () => {
     });
 
     describe('Individual Message Boxes', () => {
-      describe('Sorting on the Individual Message Inbox', () => {
-        it('individual inbox subject column', () => {
-          loginAsDocketClerk();
-          cy.get(
-            '[data-testid="message-individual-subject-header-button"]',
-          ).click();
-          verifySubjectTitleOrder({
-            boxType: 'inbox',
-            isAscending: true,
-            prefix: 'Subject Line',
-            queueType: 'individual',
-          });
-          cy.get(
-            '[data-testid="message-individual-subject-header-button"]',
-          ).click();
-          verifySubjectTitleOrder({
-            boxType: 'inbox',
-            isAscending: false,
-            prefix: 'Subject Line',
-            queueType: 'individual',
-          });
-        });
-
-        it('individual inbox received at column when defaulted to sort ascending', () => {
-          loginAsDocketClerk();
-          cy.get(
-            '[data-testid="message-individual-received-header-button"]',
-          ).click();
-          verifySubjectTitleOrder({
-            boxType: 'inbox',
-            isAscending: false,
-            prefix: 'Subject Line',
-            queueType: 'individual',
-          });
-          cy.get(
-            '[data-testid="message-individual-received-header-button"]',
-          ).click();
-          verifySubjectTitleOrder({
-            boxType: 'inbox',
-            isAscending: true,
-            prefix: 'Subject Line',
-            queueType: 'individual',
-          });
-        });
+      before(() => {
+        loginAsDocketClerk();
+      });
 
-        it('individual inbox docket number column', () => {
-          loginAsDocketClerk();
-          cy.get(
-            '[data-testid="message-individual-docket-number-header-button"]',
-          ).click();
-          verifySubjectTitleOrder({
-            boxType: 'inbox',
-            isAscending: false,
-            prefix: 'Subject Line',
-            queueType: 'individual',
-          });
-          cy.get(
-            '[data-testid="message-individual-docket-number-header-button"]',
-          ).click();
-          verifySubjectTitleOrder({
-            boxType: 'inbox',
-            isAscending: false,
-            prefix: 'Subject Line',
-            queueType: 'individual',
-          });
-        });
+      beforeEach(() => {
+        cy.visit('/');
```

Comment:
Not logging in before each sort test (we don't need too--we just need to navigate to the home screen) shaved about 15s off of this test file.